commands/addons/open.js

```diff
@@ -66,8 +66,15 @@ function * run (ctx, api) {
   if (process.env.HEROKU_SUDO) return sudo(ctx, api)
 
   let attachment = yield resolve.attachment(api, ctx.app, ctx.args.addon)
-  let webUrl
+  .catch(function (err) {
+    if (err.statusCode === 404) {
```

Comment:
@dickeyxxx updated in https://github.com/heroku/heroku-cli-addons/pull/55/commits/60ff8b523a2ed188c6d5709e7a129fe48b12c254